two-scale-heat-conduction/macro-dumux/appl/main.cc

```diff
@@ -265,6 +269,10 @@ int main(int argc, char **argv)
         xOld = x;
       }
 
+      preciceDt = couplingParticipant.getMaxTimeStepSize();
+      solverDt  = std::min(nonLinearSolver.suggestTimeStepSize(timeLoop->timeStepSize()), getParam<Scalar>("TimeLoop.MaxDt"));
```

Comment:
```suggestion
      solverDt  = std::min(nonLinearSolver.suggestTimeStepSize(timeLoop->timeStepSize()), timeLoop->maxTimeStepSize());
```
together with ```timeLoop->setMaxTimeStepSize(getParam<Scalar>("TimeLoop.MaxDt"));``` when creating the time loop.